tests/rust_usage_test/bin/monster_example.rs

```diff
@@ -1,10 +1,19 @@
 extern crate flatbuffers;
 
+#[allow(dead_code, unused_imports)]
```

Comment:
I don't think these imports are necessary in this file, please remove.

Reply:
Both these are unfortunately necessary because the monster_test.fbs schema file includes include_test1.fbs which then includes include_test2.fbs

Reply:
SGTM